### PR TITLE
Webpack testing config

### PR DESCRIPTION
**Pull request by KyEBell**
submitting pull request for DEV branch to include testing with webpack

### CODE REVIEW

**Review by kfiddle — COMMENTED** [review]
My good comment

**Review by carlosfrev123 — COMMENTED** [review]
cool

File: package.json
Comment (joie-zhang) [inline package.json:1]: Great job rearranging the dependencies!

**Review by joie-zhang — APPROVED** [review]
These changes look great! Let's do this!

**Pull request merged**